stl/inc/yvals_core.h

```diff
@@ -1403,7 +1406,21 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 
 // STL4040 is used to warn that "The contents of <any> require static RTTI."
 
-// next warning number: STL4041
+#if _HAS_CXX23 && !defined(_SILENCE_CXX23_UNIX_STREAM_DEPRECATION_WARNING) \
+    && !defined(_SILENCE_ALL_CXX23_DEPRECATION_WARNINGS)
+#define _CXX23_DEPRECATE_UNIX_STREAM                                                                                   \

```

Comment:
As LWG-3869 and https://en.wikipedia.org/wiki/STREAMS indicate, the name of the feature is apparently STREAMS plural. I think that the macros should reflect this.